test/Microsoft.AspNetCore.Mvc.FunctionalTests/Infrastructure/BasicViewsFixture.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using BasicViews;
+using Microsoft.AspNetCore.Hosting;
+
+namespace Microsoft.AspNetCore.Mvc.FunctionalTests
+{
+    public class BasicViewsFixture : MvcTestFixture<Startup>
+    {
+        protected override void ConfigureWebHost(IWebHostBuilder builder)
+        {
+            base.ConfigureWebHost(builder);
+
+            // In Production (the MvcTestFixture default), site uses asp-fallback-* attributes. The generated HTML then
```

Comment:
This should be solved now - no? https://github.com/aspnet/Mvc/blob/dev/test/Microsoft.AspNetCore.Mvc.FunctionalTests/AntiforgeryTestHelper.cs#L19